internal/dockercompose/wrapper.go

```diff
@@ -0,0 +1,135 @@
+package dockercompose
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/windmilleng/tilt/internal/logger"
+)
+
+type DockerComposeClient interface {
+	Up(ctx context.Context, pathToConfig, serviceName string, stdout, stderr io.Writer) error
+	Down(ctx context.Context, pathToConfig string, stdout, stderr io.Writer) error
+	Logs(ctx context.Context, pathToConfig, serviceName string) (io.ReadCloser, error)
+	Events(ctx context.Context, pathToConfig string) (<-chan string, error)
+	Config(ctx context.Context, pathToConfig string) (string, error)
+	Services(ctx context.Context, pathToConfig string) (string, error)
+}
+
+type cmdDCClient struct{}
+
+func NewDockerComposeClient() DockerComposeClient {
+	return &cmdDCClient{}
+}
+
+func (c *cmdDCClient) Up(ctx context.Context, pathToConfig, serviceName string, stdout, stderr io.Writer) error {
```

Comment:
seems like you could split out the fact that it always takes args `docker-compose -f <configPath>` if you wanted

Reply:
oh yo wait, just use `dcOutput`